bitnami/mlflow/templates/run/dep-job.yaml

```diff
@@ -156,8 +156,9 @@ spec:
                   key: {{ include "mlflow.v0.tracking.passwordKey" . | quote }}
             {{- end }}
             {{- end }}
-            {{- if .Values.run.extraEnvVars }}
-            {{- include "common.tplvalues.render" (dict "value" .Values.run.extraEnvVars "context" $) | nindent 12 }}
+            {{- range .Values.run.extraEnvVars }}
```

Comment:
With this change we will loose some functionality `extraEnvVars` can be set to values like this:
```yaml
  extraEnvVars:
    - name: POSTGRESQL_CLUSTER_APP_NAME
      valueFrom:
        fieldRef:
          fieldPath: metadata.name
```
Could you share more details about the problem you are trying to solve here?

Reply:
Have you faced any issue setting the `extraEnvVars`?

Reply:
I'll revert it, I saw it was not force-quoting the value, but I didn't face any specific issue with it.